From faa9265c1f27c943729ee8a8e7560cd3ed227e80 Mon Sep 17 00:00:00 2001
From: arun <arun.radhakrishnan@intel.com>
Date: Thu, 11 Feb 2016 12:57:13 -0600
Subject: [PATCH 02/15] iio:Accel: Enabling kxcjk1013 Accelerometer driver
 needed for cht-mrd

Change-Id: I54c1918798d70eda5794a20ee945d83ed636b8ff
Signed-off-by: Arun Radhakrishnan <arun.radhakrishnan@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-27733
Orig-Tracked-On: https://jira01.devtools.intel.com/browse/OAM-20023
Reviewed-on: https://android.intel.com:443/490321
---
 arch/x86/configs/x86_64_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/configs/x86_64_defconfig b/arch/x86/configs/x86_64_defconfig
index 1134800..a4ffde1 100644
--- a/arch/x86/configs/x86_64_defconfig
+++ b/arch/x86/configs/x86_64_defconfig
@@ -3654,7 +3654,7 @@ CONFIG_IIO_ST_ACCEL_3AXIS=y
 CONFIG_IIO_ST_ACCEL_I2C_3AXIS=y
 CONFIG_IIO_ST_ACCEL_SPI_3AXIS=y
 # CONFIG_KXSD9 is not set
-# CONFIG_KXCJK1013 is not set
+CONFIG_KXCJK1013=m
 
 #
 # Analog to digital converters
-- 
1.9.1

